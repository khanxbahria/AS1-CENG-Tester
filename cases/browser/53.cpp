#include "../../your_code/Window.cpp"
#include "../../your_code/Browser.cpp"
#include "../../your_code/LinkedList.h"
int main() {
{
Browser* brows = new Browser();
Window a;
Window b;
Window c;
Window d;
Tab A = Tab("k", "l","zz");
Tab B = Tab("zz", "ee","oo");
Tab C = Tab("nnn", "r","xxx");
Tab D = Tab("tt", "zz","kk");
Tab E = Tab("xxx", "ii","ddd");
Tab F = Tab("ll", "bb","z");
Tab G = Tab("rr", "y","tt");
Tab H = Tab("j", "www","i");
Tab I = Tab("aa", "c","rrr");
Tab J = Tab("aaa", "p","vv");
Tab K = Tab("kk", "ppp","p");
Tab L = Tab("iii", "xxx","tt");
Tab M = Tab("bbb", "ee","hh");
Tab N = Tab("yyy", "e","xx");
Tab O = Tab("gg", "ss","vvv");
Tab P = Tab("ww", "h","zz");
Tab Q = Tab("e", "ttt","e");
Tab R = Tab("y", "zzz","gg");
Tab S = Tab("ggg", "ooo","lll");
Tab T = Tab("qqq", "ttt","lll");
Tab U = Tab("u", "fff","v");
Tab V = Tab("q", "q","e");
Tab W = Tab("rr", "j","fff");
Tab X = Tab("k", "c","oo");
Tab Y = Tab("r", "t","tt");
Tab Z = Tab("kk", "x","b");
a.newTab(A);
b.moveActiveTabTo(3);
c.changeActiveTabTo(3);
d.changeActiveTabTo(3);
a.newTab(B);
b.changeActiveTabTo(1);
c.moveActiveTabTo(1);
d.changeActiveTabTo(1);
a.changeActiveTabTo(0);
b.moveActiveTabTo(0);
c.changeActiveTabTo(0);
d.changeActiveTabTo(0);
a.changeActiveTabTo(1);
b.moveActiveTabTo(1);
c.newTab(C);
d.changeActiveTabTo(1);
a.newTab(D);
b.moveActiveTabTo(1);
c.newTab(E);
d.moveActiveTabTo(1);
a.moveActiveTabTo(4);
b.newTab(F);
c.moveActiveTabTo(4);
d.moveActiveTabTo(4);
a.moveActiveTabTo(5);
b.newTab(G);
c.moveActiveTabTo(5);
d.changeActiveTabTo(5);
a.changeActiveTabTo(5);
b.moveActiveTabTo(5);
c.changeActiveTabTo(5);
d.changeActiveTabTo(5);
a.changeActiveTabTo(0);
b.newTab(H);
c.changeActiveTabTo(0);
d.newTab(I);
a.moveActiveTabTo(5);
b.changeActiveTabTo(5);
c.moveActiveTabTo(5);
d.moveActiveTabTo(5);
a.newTab(J);
b.newTab(K);
c.newTab(L);
d.changeActiveTabTo(4);
a.moveActiveTabTo(2);
b.moveActiveTabTo(2);
c.changeActiveTabTo(2);
d.newTab(M);
a.changeActiveTabTo(1);
b.changeActiveTabTo(1);
c.changeActiveTabTo(1);
d.newTab(N);
a.newTab(O);
b.changeActiveTabTo(0);
c.changeActiveTabTo(0);
d.newTab(P);
a.changeActiveTabTo(1);
b.newTab(Q);
c.newTab(R);
d.changeActiveTabTo(1);
a.newTab(S);
b.moveActiveTabTo(0);
c.changeActiveTabTo(0);
d.moveActiveTabTo(0);
a.newTab(T);
b.changeActiveTabTo(3);
c.moveActiveTabTo(3);
d.newTab(U);
a.moveActiveTabTo(0);
b.changeActiveTabTo(0);
c.changeActiveTabTo(0);
d.newTab(V);
a.moveActiveTabTo(1);
b.moveActiveTabTo(1);
c.changeActiveTabTo(1);
d.moveActiveTabTo(1);
a.moveActiveTabTo(1);
b.changeActiveTabTo(1);
c.changeActiveTabTo(1);
d.newTab(W);
a.changeActiveTabTo(2);
b.changeActiveTabTo(2);
c.newTab(X);
d.moveActiveTabTo(2);
a.changeActiveTabTo(4);
b.moveActiveTabTo(4);
c.changeActiveTabTo(4);
d.moveActiveTabTo(4);
a.moveActiveTabTo(4);
b.newTab(Y);
c.changeActiveTabTo(4);
d.newTab(Z);
d.changeActiveTabTo(4);
a.changeActiveTabTo(0);
b.moveActiveTabTo(0);
c.changeActiveTabTo(0);
b.changeActiveTabTo(1);
c.moveActiveTabTo(1);
b.moveActiveTabTo(0);
c.changeActiveTabTo(0);
b.changeActiveTabTo(0);
c.changeActiveTabTo(0);
d.changeActiveTabTo(0);
b.moveActiveTabTo(3);
a.moveActiveTabTo(1);
b.moveActiveTabTo(1);
c.changeActiveTabTo(1);
d.changeActiveTabTo(1);
a.changeActiveTabTo(4);
d.moveActiveTabTo(4);
a.changeActiveTabTo(2);
b.changeActiveTabTo(2);
c.changeActiveTabTo(2);
d.moveActiveTabTo(2);
a.moveActiveTabTo(3);
c.changeActiveTabTo(3);
d.moveActiveTabTo(3);
b.changeActiveTabTo(5);
c.changeActiveTabTo(5);
a.changeActiveTabTo(1);
b.changeActiveTabTo(1);
c.moveActiveTabTo(1);
d.changeActiveTabTo(1);
b.moveActiveTabTo(4);
c.changeActiveTabTo(4);
d.moveActiveTabTo(4);
a.moveActiveTabTo(3);
b.changeActiveTabTo(3);
c.changeActiveTabTo(3);
d.changeActiveTabTo(3);
a.changeActiveTabTo(1);
c.changeActiveTabTo(1);
d.changeActiveTabTo(1);
a.moveActiveTabTo(5);
b.moveActiveTabTo(5);
c.moveActiveTabTo(5);
d.changeActiveTabTo(5);
a.changeActiveTabTo(3);
b.changeActiveTabTo(3);
c.changeActiveTabTo(3);
d.moveActiveTabTo(3);
b.moveActiveTabTo(5);
c.changeActiveTabTo(5);
d.changeActiveTabTo(5);
b.changeActiveTabTo(3);
c.changeActiveTabTo(3);
a.changeActiveTabTo(2);
d.changeActiveTabTo(2);
a.changeActiveTabTo(0);
b.changeActiveTabTo(0);
c.moveActiveTabTo(0);
a.moveActiveTabTo(4);
b.changeActiveTabTo(4);
d.moveActiveTabTo(4);
c.changeActiveTabTo(5);
d.moveActiveTabTo(5);
b.moveActiveTabTo(0);
c.changeActiveTabTo(0);
d.moveActiveTabTo(0);
a.moveActiveTabTo(3);
c.changeActiveTabTo(3);
d.moveActiveTabTo(3);
a.moveActiveTabTo(2);
b.moveActiveTabTo(2);
c.changeActiveTabTo(2);
c.moveActiveTabTo(3);
d.moveActiveTabTo(3);
brows->addWindow(a);
brows->addWindow(b);
brows->addWindow(c);
brows->addWindow(d);
brows->closeEmptyWindows();
brows->getWindow(0);
brows->switchToWindow(4);
brows->print();
brows->getWindow(8);
brows->getWindow(1);
brows->newWindow();
brows->mergeAllWindows();
brows->closeWindow();
brows->closeEmptyWindows();
brows->print();
brows->closeWindow();
brows->newWindow();
brows->closeEmptyWindows();
brows->getWindow(0);
brows->getWindow(5);
brows->newWindow();
brows->print();
brows->getWindow(2);
brows->newWindow();
brows->switchToWindow(5);
brows->getWindow(8);
brows->closeEmptyWindows();
brows->getWindow(9);
brows->getWindow(5);
brows->getWindow(0);
brows->getWindow(6);
brows->getWindow(4);
brows->closeWindow();
brows->closeWindow();
brows->closeWindow();
brows->closeWindow();
brows->closeEmptyWindows();
brows->closeWindow();
brows->newWindow();
brows->closeEmptyWindows();
brows->closeWindow();
brows->getWindow(2);
brows->getWindow(8);
brows->closeWindow();
brows->getWindow(8);
brows->getWindow(2);
brows->mergeAllWindows();
brows->closeEmptyWindows();
brows->mergeAllWindows();
brows->closeEmptyWindows();
brows->getWindow(5);
brows->print();
brows->newWindow();
brows->getWindow(4);
brows->getWindow(5);
brows->closeEmptyWindows();
brows->getWindow(9);
brows->closeEmptyWindows();
brows->getWindow(5);
brows->closeEmptyWindows();
brows->newWindow();
brows->print();
brows->newWindow();
brows->print();
brows->switchToWindow(8);
brows->mergeAllWindows();
brows->closeEmptyWindows();
brows->closeEmptyWindows();
brows->mergeAllWindows();
brows->getWindow(3);
brows->closeEmptyWindows();
brows->print();
brows->getWindow(7);
brows->print();
brows->mergeAllWindows();
brows->closeEmptyWindows();
brows->newWindow();
brows->getWindow(9);
brows->mergeAllWindows();
brows->closeEmptyWindows();
brows->getWindow(0);
brows->switchToWindow(9);
brows->switchToWindow(2);
brows->closeEmptyWindows();
brows->getWindow(4);
brows->mergeAllWindows();
brows->newWindow();
brows->mergeAllWindows();
brows->switchToWindow(7);
brows->print();
brows->closeWindow();
brows->mergeAllWindows();
brows->getWindow(3);
brows->switchToWindow(6);
brows->newWindow();
brows->closeWindow();
brows->switchToWindow(5);
brows->closeWindow();
brows->closeWindow();
brows->getWindow(4);
brows->newWindow();
brows->getWindow(2);
brows->getWindow(2);
brows->getWindow(8);
brows->mergeAllWindows();
brows->closeEmptyWindows();
brows->mergeAllWindows();
brows->mergeAllWindows();
brows->getWindow(7);
brows->closeEmptyWindows();
brows->newWindow();
brows->closeWindow();
brows->mergeAllWindows();
brows->newWindow();
brows->mergeAllWindows();
brows->getWindow(2);
brows->newWindow();
brows->getWindow(8);
brows->closeEmptyWindows();
brows->print();
brows->print();
brows->mergeAllWindows();
brows->print();
brows->closeEmptyWindows();
brows->closeWindow();
brows->closeEmptyWindows();
brows->getWindow(1);
brows->mergeAllWindows();
brows->closeEmptyWindows();
delete brows;
}
	return 0;
}