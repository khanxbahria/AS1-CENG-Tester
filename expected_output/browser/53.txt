The active tab is tab 3
c
aa
rrr





x
kk
b

q
q
e

j
rr
fff

fff
u
v

ee
bbb
hh

e
yyy
xx

h
ww
zz

The active tab is tab 0




The active tab is tab 2
ee
zz
oo

l
k
zz

ttt
qqq
lll

ss
gg
vvv

p
aaa
vv

ooo
ggg
lll

zz
tt
kk





The active tab is tab 2
t
r
tt

ttt
e
e

y
rr
tt

ppp
kk
p

bb
ll
z





www
j
i

The active tab is tab 3




ii
xxx
ddd

c
k
oo

zzz
y
gg

r
nnn
xxx

xxx
iii
tt

The browser is empty
The active tab is tab 0




The active tab is tab 0




The browser is empty
The active tab is tab 0




The active tab is tab 0




The active tab is tab 0




The active tab is tab 0




The active tab is tab 0




The active tab is tab 2












The active tab is tab 2












The active tab is tab 4




















Window is empty
The list is empty.
The active tab is tab 0




The active tab is tab 1








The active tab is tab 0




The active tab is tab 1








The active tab is tab 2












Window is empty
The list is empty.
